scheduler/bin/run-docker.sh

```diff
@@ -101,6 +101,26 @@ else
     COOK_ZOOKEEPER_LOCAL=true
 fi
 
+DATA_LOCAL_IP=$(docker inspect data-local | jq -r '.[].NetworkSettings.IPAddress')
```

Comment:
How long does this take to launch? 

Run-docker takes quite a while to launch already. Can we launch more asynchronously, in parallel with e.g., the minimesos zookeeper check? Thoughts?

Reply:
In my experience, it takes about a second. Is that ok?